bambi/models.py

```diff
@@ -811,7 +815,7 @@ def predict(self, idata, kind="mean", data=None, draws=None, inplace=True):
             else:
                 X = self._design.common._evaluate_new_data(data).design_matrix
 
-        if self._design.group:
+        if self._design.group anf with_group_specific:
```

Comment:
There's a typo here, `anf` instead of `and`

Reply:
What about `include_group_specific`? I agree we need a test for this PR. 
Additionally, we should open an issue to remember adding an example of this feature.

Reply:
@alexjonesphd a test is one function (or maybe it can be a couple of functions) where you evaluate the functionality you're adding. In this case, it would be the prediction with `include_group_specific` set to False.

It would be a little harder to check the correctness of the result (i.e. not only the code does work, but the result does not contain any group-specific effect). In the meantime, I think making two predictions, one with `include_group_specific=True` and other with `include_group_specific=False` and comparing they are not the same is a good first step.

For examples about tests, see here: https://github.com/bambinos/bambi/blob/f47bda0c1eb3cc430ddbd95209c06c856751c482/bambi/tests/test_built_models.py#L455-L478

If you need help, or you simply can't do it, let us now and we will do it.

Reply:
Yes. You can push as another commit here. Once it's added we'll enable the tests to run in th PR